homeassistant/components/scene/velux.py

```diff
@@ -0,0 +1,59 @@
+"""
+Support for VELUX scenes.
+
+Connects to VELUX KLF 200 interface
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.velux/
+"""
+import logging
+
+from homeassistant.components.scene import Scene
+
+
+DEPENDENCIES = ['velux']
+_LOGGER = logging.getLogger(__name__)
+DOMAIN = "velux"
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup velux scene plattform."""
+    if DOMAIN not in hass.data \
```

Comment:
Im not sure, what exactly you suggest. Should I remove the misleading log message:

```
    if DATA_VELUX not in hass.data \
            or not hass.data[DATA_VELUX].initialized:
        return False
```

or should I just assume that the component was successfully initialized. (and then dropping all whole check?)